src/k8s_etcd.rs

```diff
@@ -176,6 +176,34 @@ impl InMemoryK8sEtcd {
         self.deleted_keys.lock().await.insert(key.to_string());
         Ok(())
     }
+
+    pub(crate) async fn update_member(&self, value: String) -> Result<()> {
+        let etcd_client = match &self.etcd_client {
+            Some(etcd_client) => etcd_client,
+            None => bail!("etcd client not configured"),
+        };
+
+        let members_list = etcd_client
+            .cluster_client()
+            .member_list()
+            .await
+            .context("during etcd members list")?;
```

Comment:
`.context` on `Result` should describe what we are doing, so `during` is a bit too verbose

```suggestion
            .context("listing etcd members")?;
```